README.md

```diff
@@ -26,22 +26,33 @@ The Apollo GraphQL extension for VS Code brings an all-in-one tooling experience
 
 <h2 id="getting-started">Getting started</h2>
 
+For the VS Code plugin to know how to find the schema, it needs to be linked to either a published schema or a local one.
+
+First, create an `apollo.config.json` file at the root of the project.
+Alternatively, you can create a `yaml`, `cjs`, `mjs`, or `ts` file with the same configuration.
+
+For the contents of this configuration file, select one of these options:
+
+<details>
+<summary>
+<h3>Setting the Extension up for use with Apollo GraphOS</h3>
+</summary>
+
 To get all the benefits of the VS Code experience, it's best to link the schema that is being developed against before installing the extension. The best way to do that is by [publishing a schema](https://www.apollographql.com/docs/graphos/delivery/publishing-schemas/) to the Apollo schema registry. After that's done:
 
-1. Create an `apollo.config.js` file at the root of the project.
-   Alternatively, you can create a `cjs`, `mjs`, or `ts` file with the same configuration.
+1. Create an `apollo.config.json` file at the root of the project.
 2. Obtain a [Personal API key](https://www.apollographql.com/docs/graphos/api-keys) from GraphOS Studio.
 
 <h3 id="apollo-config">Setting up an Apollo config</h3>
 
-For the VS Code plugin to know how to find the schema, it needs to be linked to either a published schema or a local one. To link a project to a published schema, edit the `apollo.config.js` file to look like this:
+To link a project to a published schema, edit the `apollo.config.json` file to look like this:
 
-```js
-module.exports = {
-  client: {
-    service: "my-graphql-app",
-  },
-};
+```json
+{
+  "client": {
+    "service": "graphos-graph-id"
+  }
+}
```

Comment:
I would remove this and the following `<h3>` and just have one continuous set of instructions. E.g.,

1. Obtain a personal API key.
2. Edit the `apollo.config.json` to look like this...
3. To pull down your schema, create an `.env` file ...